java/com/engflow/notificationqueue/README.md

```diff
@@ -0,0 +1,65 @@
+# Using the notification queue and event store APIs
+
+This documents explains how to make use of the **notification queue API** to
+obtain the notifications for a given build and use the notification data
+to obtain invocation events from the **event store API**. To do so,
+the [EngFlow API](https://github.com/EngFlow/engflowapis) is used as external
+dependency (see [WORKSPACE](../../../../WORKSPACE)).
+
+# Running the client
+
+In order to execute the client example, you must pass the following arguments
+that depends on the target cluster. 
+
+1. `--notification_queue_endpoint=CLUSTER_URL` holds the `grpc/grpcs` endpoint of the cluster
+2. `--queue_name=eventstore/lifecycle-events` holds the name of the queue to listen 
+3. `--tls_certificate=/path/to/your/client.crt` holds the path of the crt file to access the cluster
```

Comment:
Replace "crt file" with "certificate file containing your public key".

Add "Only needed for `grpcs://` connections."